### DISCUSSION TITLE
[Naps2.sdk installation] - Windows SDK Version - v10.0.16299 not found

### DISCUSSION THREAD

**Discussion by DaveDavePH** [cyanfish/naps2#421] [Support]
I tried to install the NAPS2.SDK on nuget manager and got an error:
![image](https://github.com/user-attachments/assets/9c6ebf68-6679-416e-a9b4-9a02d613e682)

I already download and install the windows sdk version v10.0.16299
![image](https://github.com/user-attachments/assets/52da97ac-bd5f-4492-b9b5-64155e8d740b)

but still my visual studio wont recognize the v10.0.16299
![image](https://github.com/user-attachments/assets/4de6252f-d0ea-47a9-b3fc-bd1aa8468f07)

I'm using visual studio community 2022

Thank you for your help!

**Comment by cyanfish** [comment]
Can you include the contents of your .csproj? You shouldn't need any Windows SDKs or anything like that for this to work.

**Reply by DaveDavePH** [reply]
Is this what you referring to?
![image](https://github.com/user-attachments/assets/6bd21274-8fc9-4c18-b79e-9fd0ee439e97)

Sorry I'm new to VS c#

**Reply by DaveDavePH** [reply]
I got it working! The scanner start scanning but after the scan ends it throws an error says that the scanner is busy after the scan?
![image](https://github.com/user-attachments/assets/7698965e-dc0b-409c-8971-73a3a2c86aaa)

**Reply by cyanfish** [reply]
The sample includes scanning twice (once to save images and once to save a PDF). You can delete one of them, scanning twice in a row can sometimes result in a busy message.

**Reply by DaveDavePH** [reply]
Thank you so much!
